mlflow/utils/gorilla.py

```diff
@@ -331,6 +334,41 @@ def apply(patch):
     setattr(patch.destination, _ACTIVE_PATCH, patch)
 
 
+def revert(patch):
```

Comment:
Can we add a notice to this method's docstring indicating that the method is taken from https://github.com/christophercrouzet/gorilla/blob/4dfa7215b8abacbdbd91e784da3d61d671a3a510/gorilla.py#L318-L351 with small additions for MLflow patching use cases?

Reply:
Done below